validator_test.go

```diff
@@ -310,7 +310,9 @@ func TestValidationPass(t *testing.T) {
 	for i, test := range tests {
 		t.Run(fmt.Sprintf("Test config (%v): %#v", i, test), func(t *testing.T) {
 			err := c.Unpack(test)
-			assert.NoError(t, err)
+			if err != nil {
+				t.Fatalf("test:%v error:%v", spew.Sdump(test), err)
```

Comment:
The error would be cause because `c` and `test` are "incompatible". Can you try to print c as well? Also curious how the output will look like (e.g. add a failing test).

Reply:
https://gist.github.com/alrs/e058158a9cc51f06ff2f7a6132179d0c

Reply:
Cool, I like that. Thanks!